test/runtime/runtime_test.go

```diff
@@ -22,61 +23,49 @@ var _ = Suite(&testingRuntimeSuite{})
 
 func (s *testingRuntimeSuite) TestPrependEnvsPrefix(c *C) {
 	m := []struct {
-		comment     string
-		cmd         string
-		env         map[string]string
-		soft        bool
-		expectedCmd string
-		err         string
+		comment      string
+		cmd          string
+		env          map[string]string
+		soft         bool
+		expectedCmd  string
+		expectedEnv  map[string]string
+		expectedSoft bool
+		err          string
 	}{
 		{
 			"no variable in environment",
-			"/node server.js",
-			map[string]string{},
-			false,
-			"/node server.js",
+			"/node server.js", map[string]string{}, false,
+			"/node server.js", map[string]string{}, false,
 			"",
 		},
 		{
 			"single variable in environment",
-			"/node server.js",
-			map[string]string{"PORT": "8000"},
-			false,
-			"--env=PORT=8000 /node server.js",
+			"/node server.js", map[string]string{"PORT": "8000"}, false,
+			"/node server.js", map[string]string{"PORT": "8000"}, false,
 			"",
 		},
 		{
 			"two variables in environment",
-			"/node server.js",
-			map[string]string{"PORT": "8000", "ENDPOINT": "foo.com"},
-			false,
-			// Order is not guaranteed hence the following regex is needed:
-			"(--env=PORT=8000 --env=ENDPOINT=foo.com|--env=ENDPOINT=foo.com --env=PORT=8000) /node server.js",
+			"/node server.js", map[string]string{"PORT": "8000", "ENDPOINT": "foo.com"}, false,
```

Comment:
If you use it like this, you could remove the `expectedEnv` and `expectedSoft` because they are always the same as those provided in `env` and `soft`.

I thought using explicit envs would be safer, i.e. 
```
	m := []struct {
		comment     string
		cmd         string
		env         map[string]string
		soft        bool
		expectedCmd string
		expectedEnv []string
		err         string
	}{
		{
			"two variables in environment - soft",
			"/node server.js",
			map[string]string{"PORT": "8000", "ENDPOINT": "foo.com"},
			true,
			// Order is not guaranteed hence the following regex is needed:
			"(--env=PORT\\?=8000 --env=ENDPOINT\\?=foo.com|--env=ENDPOINT\\?=foo.com --env=PORT\\?=8000) /node server.js",
			"/node server.js", map[string]string{"PORT": "8000", "ENDPOINT": "foo.com"}, true,
			[]string{"--env=PORT?=8000", "--env=ENDPOINT?=foo.com"},
			"",
		},
	}
```